hologram_dashboard/extension/src/app/dashboard.service.ts

```diff
@@ -0,0 +1,16 @@
+export class DashboardService {
+  public static API_ENDPOINT='http://localhost:8000/';
+}
+
+/** Interface for keeping track of whether a data set is available for a 
+ * specific date. */
+export interface DateAvailability {
+  date: string;
+  isAvailable: boolean;
+}
+
+/** Interface for keeping track of all the availability of a specific source. */
+export interface HologramDataAvailability {
+  sourceType: string;
+  dateAvailabilityList: Array<DateAvailability>;
+}
```

Comment:
I feel the definition of DateAvailability and HologramDataAvailability should be part of dashboard.component.ts

To be consistent with the proto, could you rename DateAvailability to AvailabilityStatus